views/index.erb

```diff
@@ -0,0 +1,24 @@
+<center>
+  <h2> Welcome to Chitter </h2>
+  <form action='/peeps'>
+    <input type='submit' value='Peeps'
+  </form>
+</center>
+
+<form action="/sessions/destroy" method="post">
+  <input type="submit" value="Sign out" />
+</form>
+
+<style>
+body {
+  height:100%;
+}
+body {
+  margin:50px;
+  font-size:100%;
+  line-height:1.5;
```

Comment:
love that you're thinking about aesthetics mate <3

Reply:
totally agrre, very nice touch given you did this prior to week 5